src/azure-cli/azure/cli/command_modules/role/_params.py

```diff
@@ -355,7 +355,7 @@ class PrincipalType(str, Enum):
                         'JSON description.')
 
     with self.argument_context('role assignment delete') as c:
-        c.argument('yes', options_list=['--yes', '-y'], action='store_true', help='Continue to delete all assignments under the subscription')
+        c.argument('yes', options_list=['--yes', '-y'], action='store_true', help='Currently no-op.')
```

Comment:
I deliberately kept `--yes` in case we want to bring https://github.com/Azure/azure-cli/pull/15277 from the dead.